forklift/drivers.py

```diff
@@ -291,6 +295,12 @@ def docker_command(self, *command, use_sshd=False):
         docker_command += [self.target]
         docker_command += command
 
+        if docker_command:
+            LOGGER.debug(
+                "docker command:\n{%s}",
```

Comment:
Don't have to do double quotes to use escaping; we're only using them for UI-facing strings, so change to single.